test/integration/install-test.js

```diff
@@ -112,6 +108,7 @@ describe(install, function() {
   });
 
   it('can install an addon with a custom default blueprint selected', async function() {
+    sinon.stub(install, 'getBlueprintNameOverride').resolves();
```

Comment:
What happens when this isn't stubbed? I like having coverage of the negative cases too, like not finding package, or package missing the package.json key.

Reply:
Yeah originally `getBlueprintNameOverride` didn't read `package.json` on disk but I had to add that since unit tests used local packages as fixtures. Removed the stubbing